docs/schedule.rst

```diff
@@ -69,3 +69,21 @@ Schedule items which are not talks have ``talk-type-none`` as the CSS class.
 
 A per item CSS class can also be set using the ``css_class`` attribute on the
 schedule item.
+
+
+Adding additional schedule validation
+=====================================
+
+Wafer runs validation on the slots and the schedule items. This behaviour
+can be extended by providing custom validators.
+
+Each validator is called with a list of all the items and is expected to
+return a list of invalid items or an empty list if the validator finds
+no error.
+
+Use ``register_schedule_item_validator`` and ``register_slot_vaildator``
+to add the validators to the list.
+
+To display the errors in the admin form, you will also need to extend the
```

Comment:
Not necessarily for this PR, but it would be cool if the default `displayerrors` block at least reported all errors so that people didn't need to override this block just to see their custom errors.